### PR TITLE
Js tweaks

### PR DESCRIPTION
**Pull request by potyl**
A few tweaks to stetho-js-rhino.

 - fix stacktrace in android m
 - fix console.log("%d", 2); // in javascript numbers are always floats

### CODE REVIEW

File: stetho-js-rhino/src/test/java/com/facebook/stetho/rhino/JsFormatTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import android.os.Build;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.junit.Assert.*;
+
+@Config(emulateSdk = Build.VERSION_CODES.JELLY_BEAN)
+@RunWith(RobolectricTestRunner.class)
```
Comment (jasta) [inline stetho-js-rhino/src/test/java/com/facebook/stetho/rhino/JsFormatTest.java]: Do you really need this test to use Robolectric?  It seems quite like this should be able to survive with just the standard JRE.

**Comment by jasta** [comment]
lgtm, feel free to cleanup/squash as you see fit and I'll merge.

File: stetho-js-rhino/src/test/java/com/facebook/stetho/rhino/JsFormatTest.java (hunk 1 above)
Comment (potyl) [inline stetho-js-rhino/src/test/java/com/facebook/stetho/rhino/JsFormatTest.java]: Fixing it. I just copied the existing unit tests steth-okhttp and built on top.

**Comment by potyl** [comment]
Unit tests running with junit only and squashed.

**Comment by jasta** [comment]
Thanks as always :)

**Comment by potyl** [comment]
I just noticed that in master roboelectric is still used!

**Pull request merged**